src/components/nav-options/index.tsx

```diff
@@ -36,7 +36,7 @@ const NavOptions = ({ options }: NavOptionsProps) => {
             </Button>
           ) : (
             <Button
-              className="p-2"
+              className="p-2 whitespace-normal"
```

Comment:
The "public profile" nav text can be trimmed. 

Resolutions to test:
- iPhone 12/13 Pro: 390x844 
- iPhone 11 Pro: 375x812

![Screen Shot 2025-02-15 at 12 39 08](https://github.com/user-attachments/assets/ddb389ae-5298-4244-a810-a950f6760406)